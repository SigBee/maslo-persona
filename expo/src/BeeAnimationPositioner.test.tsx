import React from 'react';
import { describe, expect, it } from '@jest/globals';
import renderer from 'react-test-renderer';

import {BeeAnimationPositioner} from './BeeAnimationPositioner';

jest.useFakeTimers();

describe('BeeAnimationPositioner', () => {

    it('Check if it renders correctly', () => {
        const tree = renderer.create(<BeeAnimationPositioner  nextPosition={{ x: 0, y: 0 }} children={undefined} />).toJSON();
        expect(tree).toMatchSnapshot();
    });

});