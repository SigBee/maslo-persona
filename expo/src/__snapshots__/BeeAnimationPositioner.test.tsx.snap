// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BeeAnimationPositioner Check if it renders correctly 1`] = `
<View
  collapsable={false}
  style={
    Object {
      "backgroundColor": "transparent",
      "height": 375,
      "left": 375,
      "position": "absolute",
      "top": 667,
      "transform": Array [
        Object {
          "translateX": -187.5,
        },
        Object {
          "translateY": -428.57142857142856,
        },
      ],
      "width": 375,
    }
  }
/>
`;
